### ISSUE TITLE
New warnings from IPython terminal

### ISSUE THREAD

**Issue by astrofrog** [glue-viz/glue#2005]
I've started noticing the following warnings/errors coming from IPython, so this needs investigating:

```
Call stack:
  File "/Users/tom/python/pyside2/lib/python3.7/site-packages/IPython/core/interactiveshell.py", line 3661, in atexit_operations
    self.reset(new_session=False)
  File "/Users/tom/python/pyside2/lib/python3.7/site-packages/IPython/core/interactiveshell.py", line 1449, in reset
    self.alias_manager.soft_define_alias(cmd, cmd)
  File "/Users/tom/python/pyside2/lib/python3.7/site-packages/IPython/core/alias.py", line 221, in soft_define_alias
    error("Invalid alias: %s" % e)
Message: "Invalid alias: The name man can't be aliased because it is another magic command."
Arguments: ()
```

**Comment by astrofrog** [comment]
See https://github.com/ipython/ipython/issues/11646

**Comment by astrofrog** [comment]
Fixed in ipython 7.7
